Src/Witsml/Data/WitsmlWell.cs

```diff
@@ -1,16 +1,30 @@
 using System.Collections.Generic;
 using System.Xml.Serialization;
 
+using Witsml.Data.Measures;
+
 namespace Witsml.Data
 {
     public class WellDatum
     {
+        [XmlAttribute("uid")]
+        public string Uid { get; set; }
         [XmlElement("name")]
         public string Name { get; set; }
         [XmlElement("code")]
         public string Code { get; set; }
         [XmlElement("elevation")]
-        public string Elevation { get; set; }
+        public WellElevationCoord Elevation { get; set; }
```

Comment:
I think WitsmlMeasureWithDatum can be used here, given that it has the same fields as WellElevationCoord.

Reply:
Yes, you're right.

Looking at the schema, I see measuredDepthCoord, wellVerticalDepthCoord, and wellElevationCoord have the same fields, but the difference between them is that the allowed values for uom are slightly different. Since it doesn't seem like there is any need to restrict that uom to an enumeration in code too, then yeah WitsmlMeasureWithDatum makes sense.

That said, looking at the code, I see there is a WitsmlMeasuredDepthCoord class that inherits from WitsmlMeasureWithDatum, but doesn't add any new members. This class is used many places in the code. Should a similar subclassing be done here for WellElevationCoord, or just use WitsmlMeasureWithDatum directly?

Reply:
I do not think there is any benefit to using Measure classes that do not add new members, given that we do not have any plans to handle the different kinds of measures based on the derived types. I assume that WitsmlMeasureWithDatum should be sufficient for now. As long as the required fields are present in the class it should be fine either way.